maestro/src/main/java/org/intocps/maestro/Main.java

```diff
@@ -181,27 +180,37 @@ public static void main(String[] args) throws Exception {
             new MableInterpreter(new DefaultExternalValueFactory(workingDirectory,
                     IOUtils.toInputStream(mabl.getRuntimeDataAsJsonString(), StandardCharsets.UTF_8))).execute(mabl.getMainSimulationUnit());
         }
+        return true;
+    }
+
+    public static void main(String[] args) throws Exception {
+        int exitCode = 0;
```

Comment:
so we always exit 1 no matter the return?

Reply:
haha ups!